tensorboard/main.py

```diff
@@ -114,6 +114,15 @@
     'event_file', '',
     'The particular event file to query for. Only used if --inspect is present '
     'and --logdir is not specified.')
+tf.flags.DEFINE_string(
+    'path_prefix', '',
+    'An optional, relative prefix to the path, e.g. "/path/to/tensorboard". '
+    'resulting in the new base url being located at '
+    'localhost:6006/path/to/tensorboard under default settings. A leading '
+    'slash is required when specifying the path_prefix, however trailing '
+    'slashes can be omitted. The path_prefix can be leveraged for path '
+    'based routing of an elb when the website base_url is not available '
```

Comment:
What's an elb? Maybe proxy instead?

Reply:
Elastic load balancer.

Googlebot says

> Note to project maintainer: This is a terminal state, meaning the cla/google commit status will not change from this State. It's up to you to confirm consent of the commit author(s) and merge this pull request when appropriate.

so I think we don't need to do anything further.

Reply:
Hmm, @jart, are you the project maintainer? :)